src/V3AssertPre.cpp

```diff
@@ -139,16 +152,195 @@ class AssertPreVisitor final : public VNVisitor {
     // VISITORS
     //========== Statements
     void visit(AstClocking* nodep) override {
+        m_clockingp = nodep;
         UINFO(8, "   CLOCKING" << nodep << endl);
-        // Store the new default clock, reset on new module
-        m_seniDefaultp = nodep->sensesp();
-        // Trash it, keeping children
-        if (nodep->bodysp()) {
-            nodep->replaceWith(nodep->bodysp()->unlinkFrBack());
+        iterateChildren(nodep);
+    }
+    void visit(AstClockingItem* nodep) override {
+        auto* const flp = nodep->fileline();
```

Comment:
```suggestion
        FileLine* const flp = nodep->fileline();
```

When type is "obvious" please avoid auto.